src/components/Queue.vue

```diff
@@ -83,7 +83,7 @@ export default {
                 // so that we can reflect any changes in the Queue component.
                 var queueState = val.sendingStatus;
                 if(!queueState && val.sentData) {
-                    var message = this.genQueueSuccessMessage(val.sentData);
+                    var message = this.queueSuccessMessage(val.sentData);
```

Comment:
Do we really need this method anymore if it's just returning val.sentData.message?